docs/running-rules.md

```diff
@@ -77,27 +86,26 @@ Create a YAML rule file that you can run repeatedly.
       pattern: $SOMEVAR is "..."
       severity: ERROR
     ```
-    See [Getting started](../writing-rules/overview/) for the full example.
 3. Run the following command to run local YAML rule files:
     ```sh
     semgrep --config path/to/rule.yaml
     ```
 
-### Running multiple rules simultaneously
-
-To run multiple rules simultaneously, use `--config` before every YAML URL, or Semgrep registry entry name. See the following code example (substitute the colored values as necessary):
+For more information, see [Getting started](../writing-rules/overview/).
 
-<pre class="language-bash"><code>semgrep --config <span className="placeholder">p/python</span> --config <span className="placeholder">myrules/myrule.yaml</span></code></pre>
+## Running multiple rules simultaneously
 
-## Appendix
+To run multiple rules simultaneously, use `--config` before every YAML URL, or Semgrep registry entry name. This option let's you include your local rules as well as Semgrep Registry rules. See the following code example (substitute the colored values as necessary):
 
-We are working on optimizations to improve Semgrep's performance, which necessitates a change in rules processing. If you are using v0.55.0 or later and encounter an unexpected metavariable binding or missing result, try running with `--optimizations none` to use the original code path. 
-
-You may find some files that Semgrep previously parsed are now skipped; this happens when Semgrep can confirm the rule does not match the file without parsing it. You can similarly run Semgrep with `--optimizations none` to avoid this behavior.
+<pre class="language-bash"><code>semgrep --config <span className="placeholder">p/python</span> --config <span className="placeholder">MYRULES/MYRULE.YAML</span></code></pre>
 
 ## Findings
 
 * See [Managing findings](../managing-findings/) for information on Semgrep findings.
 * See [Ignoring findings](../ignoring-files-folders-code/) for details on suppressing rule output.
 
+## Next steps
+
+Contribute to the registry by writing your own rules and adding them to the <a href="https://github.com/returntocorp/semgrep-rules" target="_blank">Semgrep rules repository</a>.
```

Comment:
nice!

Reply:
didn't you have a docs page on how to contribute to the registry?

Reply:
Changing to:

```
Contribute to the registry by writing your own rules and adding them to the <a href="https://github.com/returntocorp/semgrep-rules" target="_blank">Semgrep rules repository</a>. Find out how to contribute to Semgrep rules repository by reading [Contributing to Semgrep rules](contributing/contributing-rules.md) guide.
```